source/administration/devops-command-center.rst

```diff
@@ -62,50 +62,124 @@ The set of steps contained within a playbook is called a checklist.
 Viewing and creating playbooks
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Select **!** in the channel header to open the RHS panel. Select the **Playbooks** icon to open the playbooks backstage and list all playbooks associated with the current team. Select **+ New Playbook** to start a new playbook and checklist. Enter a playbook name and choose **Save Playbook**.
+Open **Main Menu > Incidents & Playbooks > Playbooks** to open the playbooks backstage and view the public playbooks associated with the current team. Select **+ New Playbook** to start a new playbook.
 
-Creating checklists
-~~~~~~~~~~~~~~~~~~~
+Public and private incidents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Checklists are created within a playbook and provide steps to follow during an incident. Checklists can be created ahead of time as part of a playbook, or during an incident.
+Incident channels can be public or private, based on the settings of the playbook used for the incident.
 
-Editing checklists
-~~~~~~~~~~~~~~~~~~~
+- **Public:** Anyone in the team can view the incident details and join the incident channel. The incident details are visible to all team members during the active incident and after the incident has ended.
+- **Private:** Only incident members who were members of the channel when the incident started have access to the ongoing incident and incident details after the incident has ended. Private incidents can't be made public once they've been started.
+
+Private incidents don't appear in the backstage when they're active, and the contents of the channel won't show up in a search query. If an incident was public and changed to private, it can't be accessed by non-participants unless they're invited by a member of the incident channel.
+
+**Creating a public playbook and incident**
+
+1. Navigate to **Main Menu > Incidents & Playbooks > Playbooks**.
+2. Select **+ New Playbook**.
+3. Enter a name for the playbook.
+4. Move the toggle to the right to **Create Public Incident**.
+5. Add a few checklist items.
+6. Select **Save**.
+7. Switch to the main channel view.
+8. Create an incident by selecting the playbook you just created.
 
-You can add steps and edit checklist items during an active incident. You can also edit checklist items in playbooks that are not being used in active incidents.
+**Creating a private playbook and incident**
 
-To edit a checklist, choose **Edit**.
+1. Navigate to **Main Menu > Incidents & Playbooks > Playbooks**.
+2. Select **+ New Playbook**.
+3. Enter a name for the playbook.
+4. Keep the **Create Public Incident** option disabled.
+5. Add a few checklist items.
+6. Select **Save**.
+7. Switch to the main channel view.
+8. Create an incident by selecting the playbook you just created.
 
-- Select the text field to edit the item details.
-- Hover over an input box and select the **X** to remove the checklist item.
-- Drag the hamburger menu to rearrange the checklist items.
+Editing checklists
```

Comment:
Checklists within an incident are now immutable, so we can remove this section altogether.